Calendars.nuspec

```diff
@@ -80,8 +80,10 @@ Android all-day events are no longer affected by timezone.
      <!--Tizen-->
      <file src="Calendars\Calendars.Plugin.Tizen\bin\Release\tizen40\Plugin.Calendars.dll" target="lib\tizen40\Plugin.Calendars.dll" />
      <file src="Calendars\Calendars.Plugin.Tizen\bin\Release\tizen40\Plugin.Calendars.pdb" target="lib\tizen40\Plugin.Calendars.pdb" />
-     <file src="Calendars\Calendars.Plugin.Abstractions\bin\Release\Plugin.Calendars.Abstractions.dll" target="lib\tizen40\Plugin.Calendars.Abstractions.dll" />
-     <file src="Calendars\Calendars.Plugin.Abstractions\bin\Release\Plugin.Calendars.Abstractions.pdb" target="lib\tizen40\Plugin.Calendars.Abstractions.pdb" />
+     <file src="Calendars\Calendars.Plugin.Tizen\bin\Release\tizen40\Plugin.Calendars.xml" target="lib\tizen40\Plugin.Calendars.xml" />
+     <file src="Calendars\Calendars.Plugin.Abstractions\bin\Release\netstandard1.0\Plugin.Calendars.Abstractions.dll" target="lib\tizen40\Plugin.Calendars.Abstractions.dll" />
+     <file src="Calendars\Calendars.Plugin.Abstractions\bin\Release\netstandard1.0\Plugin.Calendars.Abstractions.pdb" target="lib\tizen40\Plugin.Calendars.Abstractions.pdb" />
+     <file src="Calendars\Calendars.Plugin.Abstractions\bin\Release\netstandard1.0\Plugin.Calendars.Abstractions.xml" target="lib\tizen40\Plugin.Calendars.Abstractions.xml" />
```

Comment:
Good catch on the netstandard stuff, but this is still failing on appveyor because tizen40\Plugin.Calendars.pdb doesn't exist